src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -1,226 +1,90 @@
-      const enterpriseConfig = {
-        uuid: 'test-enterprise-uuid',
-      };
-      const tree = renderer
-        .create((
-          <AppContext.Provider value={{ enterpriseConfig }}>
-            <CourseEnrollments
-              {...initialProps}
-              isMarkCourseCompleteSuccess
-            />
-          </AppContext.Provider>
-        ))
-        .toJSON();
-      expect(tree).toMatchSnapshot();
-    });
-  });
-
-  describe('sidebar', () => {
-    let wrapper;
-
-    it('is not shown at screen widths greater than or equal to large breakpoint', () => {
-      global.innerWidth = breakpoints.large.minWidth;
-      const enterpriseConfig = {
-        uuid: 'test-enterprise-uuid',
-      };
-      wrapper = mount((
-        <AppContext.Provider value={{ enterpriseConfig }}>
-          <CourseEnrollments {...initialProps} />
-        </AppContext.Provider>
-      ));
-      expect(wrapper.find('.sidebar-example').exists()).toBeFalsy();
-    });
-
-    it('is shown at screen widths less than large breakpoint', () => {
-      global.innerWidth = breakpoints.small.minWidth;
-      const enterpriseConfig = {
-        uuid: 'test-enterprise-uuid',
-      };
-      wrapper = mount((
-        <AppContext.Provider value={{ enterpriseConfig }}>
-          <CourseEnrollments {...initialProps} />
-        </AppContext.Provider>
-      ));
-      expect(wrapper.find('.sidebar-example').exists()).toBeTruthy();
-    });
-  });
+jest.mock('../course-cards/mark-complete-modal/data/service');
 
-  describe('calls appropriate fetch method depending on page type', () => {
-    beforeEach(() => {
-      mockFetchCourseEnrollments.mockReset();
-      mockClearCourseEnrollments.mockReset();
-    });
-
-    it('for enterprise page', () => {
-      const uuid = 'test-enterprise-uuid';
-      const enterpriseConfig = {
-        uuid,
-      };
-      mount((
-        <AppContext.Provider value={{ enterpriseConfig }}>
-          <CourseEnrollments {...initialProps} />
-        </AppContext.Provider>
-      ));
-      expect(mockFetchCourseEnrollments.mock.calls.length).toEqual(1);
-      expect(mockFetchCourseEnrollments).toBeCalledWith({
-        uuid,
-      });
-    });
-  });
+beforeEach(() => {
+  updateCourseCompleteStatusRequest.mockImplementation(() => {});
+});
 
-  it('properly closes mark course as complete success status alert', () => {
-    const enterpriseConfig = {
-      uuid: 'test-enterprise-uuid',
-    };
-    const wrapper = mount((
+const genericMockFn = () => jest.fn();
+const store = createMockStore(configureMockStore);
+
+const enterpriseConfig = {
+  uuid: 'test-enterprise-uuid',
+};
+const completedCourseRun = createCompletedCourseRun();
+const inProgCourseRun = { ...completedCourseRun, courseRunStatus: 'in_progress' };
+
+const defaultInitialProps = defaultInitialEnrollmentProps({ genericMockFn });
+
+// TODO not sure why and if we need the sidebarComponent here
```

Comment:
Back when the UX team gave us the original wireframes/mocks for this page, there was a requirement for small screens (aka mobile) to show the sidebar below the "My courses in progress" section but above the "Upcoming"/"Archived" sections (as opposed to having the sidebar rendered after all the course sections). To be honest, I'm not really sure what the rationale for that was but I just pinged Brandon C. (UX) to get clarification if this is still a requirement.

If it isn't, we could stop rendering the `Sidebar` within this `CourseEnrollments` section. I will follow-up with you once I hear back from Brandon C about this, but for now, yeah... `Sidebar` gets rendered within `CourseEnrollments` for mobile [between the in-progress and upcoming/archived sections](https://github.com/edx/frontend-app-learner-portal-enterprise/blob/master/src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx#L124).

Reply:
awesome! that is great context and help, we can and should do any actions as necessary after this merges and update the tests too thanks